docs/how-to-build-with-ddn/with-oracle.mdx

```diff
@@ -0,0 +1,431 @@
+
+```sh title="Before you can create a new Hasura DDN project, you need to authenticate your CLI:"
+ddn auth login
+```
+
+This will launch a browser window prompting you to log in or sign up for Hasura DDN. After you log in, the CLI will
+acknowledge your login, giving you access to Hasura Cloud resources.
+
+### Step 2. Scaffold out a new local project
+
+```sh title="Next, create a new local project:"
+ddn supergraph init my-project && cd my-project
+```
+
+Once you move into this directory, you'll see your project scaffolded out for you. You can view the structure by either
+running `ls` in your terminal, or by opening the directory in your preferred editor.
+
+### Step 3. Initialize your Oracle connector
+
+```sh title="In your project directory, run:"
+ddn connector init my_oracle -i
+```
+
+From the dropdown, start typing `oracle` and hit enter to accept the default port. Then, provide the following values:
+
+**JDBC URL**
+
+```plaintext
+jdbc:oracle:thin:@local.hasura.dev:1521/XEPDB1?user=example&password=mypassword
+```
+
+For Schemas, hit enter to use the database provided in the JDBC URL.
+
+### Step 4. Start the local Oracle container
+
+Run the following command to start the Oracle container:
+
+```sh
+docker run -d \
+  --name oracle-server \
+  --restart always \
+  -p 1521:1521 \
+  -e ORACLE_PASSWORD=oraclepassword \
+  -e APP_USER=example \
+  -e APP_USER_PASSWORD=mypassword \
+  -e TARGET_PDB=XEPDB1 \
+  gvenzl/oracle-xe:21.3.0-slim
+```
+
+```sh title="Use the SQL*PLUS shell to start the SQL prompt:"
+docker exec -it oracle-server sqlplus example/mypassword@local.hasura.dev:1521/XEPDB1
+```
+
+```SQL title="Create a table in the database:"
+CREATE TABLE users (user_id Number NOT NULL, name Varchar2(45) NOT NULL, age Number NOT NULL);
+```
+
+```SQL title="Then, seed the table:"
+INSERT ALL
+   INTO users (user_id, name, age) VALUES (1, 'Alice', 25)
+   INTO users (user_id, name, age) VALUES (2, 'Bob', 30)
+   INTO users (user_id, name, age) VALUES (3, 'Charlie', 35)
+SELECT 1 FROM DUAL;
+```
+
+```SQL title="You can verify this by running:"
+SELECT * FROM users;
+```
+
+You should see a list of users returned.
+
+### Step 5. Introspect your Oracle database
+
+```sh title="Next, use the CLI to introspect your Oracle database:"
+ddn connector introspect my_oracle
+```
+
+After running this, you should see a representation of your database's schema in the
+`app/connector/my_oracle/configuration.json` file; you can view this using `cat` or open the file in your editor.
+
+```sh title="Additionally, you can check which resources are available — and their status — at any point using the CLI:"
+ddn connector show-resources my_oracle
+```
+
+### Step 6. Add your model
+
+```sh title="Now, track the table from your Oracle database as a model in your DDN metadata:"
+ddn models add my_oracle "EXAMPLE.USERS"
+```
+
+Open the `app/metadata` directory and you'll find a newly-generated file: `Users.hml`. The DDN CLI will use this Hasura
```

Comment:
```suggestion
Open the `app/metadata` directory and you'll find a newly-generated file: `ExampleUsers.hml`. The DDN CLI will use this Hasura
```

I'm assuming this would be correct?